### DISCUSSION TITLE
How to make sure this

### DISCUSSION THREAD

**Discussion by shyjuu** [aclap-dev/video-downloadhelper#1186] [Help and Q&A]
### Your question

I want to download videos and downloaded videos should be with file name 1 , 2 ,3 , I mean it should be named as we download videos 

### Addon & CoApp version and details

I want to download videos and downloaded videos should be with file name 1 , 2 ,3 , I mean it should be named as we download videos

**Comment by Mattmjs** [comment]
You can name them however you want by using
**download as** which provides a dialog before the download starts.
